src/draw/lv_draw_img.c

```diff
@@ -182,39 +182,6 @@ bool lv_img_cf_has_alpha(lv_img_cf_t cf)
     return has_alpha;
 }
 
-/**
- * Get the type of an image source
- * @param src pointer to an image source:
- *  - pointer to an 'lv_img_t' variable (image stored internally and compiled into the code)
- *  - a path to a file (e.g. "S:/folder/image.bin")
- *  - or a symbol (e.g. LV_SYMBOL_CLOSE)
- * @return type of the image source LV_IMG_SRC_VARIABLE/FILE/SYMBOL/UNKNOWN
- */
-lv_img_src_t lv_img_src_get_type(const void * src)
```

Comment:
Since there's the new `lv_img_src_t` type and the matching lv_img_src.c/h files, it's where it's gone. Everything related to image source is in this header/source file now.

Reply:
My bad, sorry. I though it was moved to `lv_img.c`.